hoomd/mpcd/CollisionMethod.h

```diff
@@ -103,9 +107,27 @@ class PYBIND11_EXPORT CollisionMethod : public Autotuned
     uint64_t m_period;        //!< Number of timesteps between collisions
     uint64_t m_next_timestep; //!< Timestep next collision should be performed
 
+    bool m_checked_collision_warnings; //!< True if collision related warnings have been checked
+
+    GPUArray<Scalar4> m_initial_velocity; //!< Initial velocities of the embedded particles
+    GPUArray<Scalar3> m_linmom_accum;     //!< Accumulated change in linear momentum of rigid bodies
+    GPUArray<Scalar4> m_angmom_accum; //!< Accumulated change in angular momentum of rigid bodies
+
     //! Check if a collision should occur and advance the timestep counter
     virtual bool shouldCollide(uint64_t timestep);
 
+    //! Begin process of applying collisions to rigid bodies
+    virtual void beginRigidBodyCollision(uint64_t timestep);
```

Comment:
Now that we have multiple methods, let's make them a bit more descriptive:
```suggestion
    virtual void storeInitialEmbeddedGroupVelocities(uint64_t timestep);
```